pkg/api/v1/pod/util.go

```diff
@@ -395,3 +395,20 @@ func UpdatePodCondition(status *v1.PodStatus, condition *v1.PodCondition) bool {
 	// Return true if one of the fields have changed.
 	return !isEqual
 }
+
+// resourcesEqual returns true if the two resources lists provided are equal.
+func resourcesEqual(lhs v1.ResourceList, rhs v1.ResourceList) bool {
```

Comment:
Can we drop this from the first commit since it is moved to a different file in the second commit?

Reply:
Yup, will do.